docs/concepts/orchestration/flow-placeholder.md

```diff
@@ -0,0 +1,1099 @@
+(flow-cookbook)=
+# Flow
+
+```{important}
+A Flow is a kind of {class}`~jina.Orchestration`. Be sure to read up on that too!
+```
+
+A {class}`~jina.Flow` orchestrates {class}`~jina.Executor`s into a processing pipeline to accomplish a task. Documents "flow" through the pipeline and are processed by Executors.
```

Comment:
A `Flow` should be described in relationship to `Deployments`. At the end a `Flow` is a set of `Deployments`

Reply:
Resolved in #10517927